cmd/nerdctl/namespace.go

```diff
@@ -84,10 +94,11 @@ func namespaceLsAction(cmd *cobra.Command, args []string) error {
 
 	w := tabwriter.NewWriter(cmd.OutOrStdout(), 4, 8, 4, ' ', 0)
 	// no "NETWORKS", because networks are global objects
-	fmt.Fprintln(w, "NAME\tCONTAINERS\tIMAGES\tVOLUMES")
+	fmt.Fprintln(w, "NAME\tCONTAINERS\tIMAGES\tVOLUMES\tLABELS")
```

Comment:
Labels should be shown in `nerdctl namespace inspect`, not `nerdctl namespace ls`

Reply:
I refer to "ctr ns ls",do you think is it necessary to add "nerdctl ns inspect" command?
namespace doesn't have many useful message

Reply:
Yes, for consistency with other commands

Reply:
I will add it tomorrow